Source/dafny.msbuild/build/dafny.msbuild.props

```diff
@@ -16,11 +16,19 @@
          -->
         <NoWarn>0164;0219;1717;0162;0168</NoWarn>
 
-        <!-- 
+        <!--
         Defaulting to an empty string means we assume that the Dafny
         Binaries directory is already on your $PATH
         -->
         <DafnyBinariesDir></DafnyBinariesDir>
+
+        <!-- Optional, mutually-exclusive properties for Dafny-lang CLI version checking. -->
+        <!--suppress CheckTagEmptyBody -->
```

Comment:
I wonder why this isn't necessary for `<DafnyBinariesDir>` above?

Reply:
Oops, it is. Fixed in latest commits.